conf/messages

```diff
@@ -52,4 +52,31 @@ overview.heading.isAgent.true = Your client’s Income Tax Return
 overview.heading.isAgent.false = Your Income Tax Return
 
 overview.notStarted = Not started
-overview.updated = Updated
\ No newline at end of file
+overview.updated = Updated
+
+errors.500.heading = Sorry, there is a problem with the service
+errors.500.p1 = Try again later.
+errors.500.p2 = You can also:
+errors.500.p3 = go to the
+errors.500.p4 = Income Tax home page (opens in new tab)
+errors.500.p5 = for more information
+errors.500.p6 = use
+errors.500.p7 = Self Assessment: general enquiries (opens in new tab)
+errors.500.p8 = to speak to someone about your income tax
+
+errors.404.heading = Page not found
+errors.404.p1 = If you typed the web address, check it is correct.
+errors.404.p2 = If you used ‘copy and paste’ to enter the web address, check you copied the full address.
+errors.404.p3 = If the website address is correct or you selected a link or button, you can use
+errors.404.p4 = Self Assessment: general enquiries (opens in new tab)
+errors.404.p5 = to speak to someone about your income tax.
```

Comment:
This is inconsistent with the final line of the other error pages (full stop)

Reply:
This is okay as the other links are in bullet points (https://design-system.service.gov.uk/styles/typography/#bulleted-lists)